qiita_db/support_files/patches/20.sql

```diff
@@ -0,0 +1,6 @@
+-- March 19, 2015
+-- Rename columns to be more descriptive and allow easier joins
+ALTER TABLE qiita.processed_data_status RENAME COLUMN description TO status_description;
```

Comment:
`processed_data_status_description` so future changes do not break this (i.e. there are other status tables, and if they eventually get added, it will break this...)